### PR TITLE
Informational fixes

### PR DESCRIPTION
**Pull request by ilikesymmetry**

### CODE REVIEW

File: src/PermissionManager.sol [hunk 1]
```diff
@@ -240,8 +240,6 @@ contract PermissionManager is IERC1271, Ownable2Step, Pausable {
 
     /// @notice Revoke a permission to disable its use indefinitely.
     ///
-    /// @dev Depending on permission contract implementation, permissions can revoke other permissions.
-    ///
     /// @param permissionHash hash of the permission to revoke
```
Comment (darian-chan-cb) [inline src/PermissionManager.sol:243]: Finding #4 recommended to add a note here that says something along the lines of "Once a permission is revoked, it cannot be re-enabled. A new permission will need to be approved."

should we address it here for clarity?

Comment (darian-chan-cb) [reply src/PermissionManager.sol:243]: I saw the comment was added in the docs, rather than at the revokePermission() function, I'll resolve this comment

**Review by darian-chan-cb — APPROVED** [review]

**Comment by darian-chan-cb** [comment]
I also saw finding #14 wasn't fixed (gas saving by changing the logic of `_isValidInitialization`). I think it's ok to leave it unchanged because we wouldn't want users initializing with a `start = 0` or `period = 0` anyways, so it's nice to have that check in there

File: src/mixins/NativeTokenRecurringAllowance.sol [hunk 2]
```diff
@@ -200,7 +200,7 @@ abstract contract NativeTokenRecurringAllowance {
         CycleUsage memory lastCycleUsage = _lastCycleUsages[account][permissionHash];
 
         // last cycle exists if start, end, and spend are non-zero
```
Comment (darian-chan-cb) [inline src/mixins/NativeTokenRecurringAllowance.sol:202]:
```suggestion
        // last cycle exists if start, end, and spend are non-zero, i.e. a non-zero start implies that end and spend are also non-zero
```

**Comment by chinmay-farkya** [comment]
> I also saw finding `14` wasn't fixed (gas saving by changing the logic of `_isValidInitialization`). I think it's ok to leave it unchanged because we wouldn't want users initializing with a `start = 0` or `period = 0` anyways, so it's nice to have that check in there

Hey @darian-chan-cb finding 14 does not say that the logic inside _isValidInitialization be changed, what I meant was this => At the particular spot in code only one check is sufficient ( _isValidInitialization actually checks two things), so you can replace if(! _isValidInitialization) with if(! savedRecurringAllowance.start > 0)

**Pull request merged**